sshproxy-managerd/sshproxy-managerd.go

```diff
@@ -203,6 +204,17 @@ func genKey(user, host string) string {
 	return fmt.Sprintf("%s@%s", user, host)
 }
 
+// getUserFromKey returns the user from the key used in the proxiedConnections global variable.
+func getUserFromKey(key string) (string, error) {
+	re := regexp.MustCompile(`^(\w*)@`)
+	match := re.FindStringSubmatch(key)
```

Comment:
I would prefer to have the two lines merged i.e. `match := regexp.MustCompile(...).FindStringSubmatch(key)`.